lefthk-core/src/errors.rs

```diff
@@ -1,7 +1,7 @@
 use thiserror::Error;
 
 macro_rules! log_on_error {
-    ($a: expr) => {
+    ($a: expr_2021) => {
```

Comment:
Maybe we want to add a `TODO` to eventually fully update this to edition 2024?

Reply:
Good idea! Added